src/components/virtualRepeat/virtual-repeater.js

```diff
@@ -620,7 +620,7 @@ VirtualRepeatController.prototype.containerUpdated = function() {
         this.repeatListExpression,
         angular.bind(this, function(items) {
           if (items && items.length) {
-            this.$$rAF(angular.bind(this, this.readItemSize_));
+            this.readItemSize_();
```

Comment:
Actually with rAF, it reads the `itemSize` correctly the first time, whereas without it, the first time is 0 and the next time comes back proper, presumably because the browser isn't done laying the element out.

That being said, I really have no idea why this helps. Here's a quote from MDN:

> The window.requestAnimationFrame() method tells the browser that you wish to perform an animation and requests that the browser call a specified function to update an animation **before the next repaint**.

My best guess is that sometimes that next repaint might take a long time, which is why the browser delays the execution until it does, causing it to hang.